File: src/main/java/no/nav/sbl/dialogarena/modiasyforest/rest/domain/adresse/PostboksadresseNorsk.java
package no.nav.sbl.dialogarena.modiasyforest.rest.domain.adresse;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class PostboksadresseNorsk extends StrukturertAdresse {
    public String postnummer;
    public String postboksanlegg;
    private String postboksnummer;
}

File: src/main/kotlin/no/nav/syfo/controller/user/domain/adresse/PostboksadresseNorsk.kt
package no.nav.syfo.controller.user.domain.adresse

class PostboksadresseNorsk(
    val poststed: String? = null,
    val postboksanlegg: String? = null,
    private val postboksnummer: String? = null
) : StrukturertAdresse()
